app/views/shared/_maintenance_window_alert.html.erb

```diff
@@ -0,0 +1,26 @@
+<%
+  maintenance_window = MaintenanceWindow.new(
+    start: Figaro.env.acuant_maintenance_window_start,
+    finish: Figaro.env.acuant_maintenance_window_finish,
+    now: local_assigns[:now],
+  )
+%>
+<% if maintenance_window.active? %>
+  <div class="usa-alert usa-alert--warning margin-bottom-2">
+    <div class="usa-alert__body">
+      <div class="usa-alert__text">
+        <p>
+          <%= t('notices.maintenance.currently_under_maintenance_html',
+                finish: l(maintenance_window.finish,
+                          format: t('time.formats.event_timestamp_with_zone'))) %>
+        </p>
+        <p class="margin-bottom-0">
+          <%= t('notices.maintenance.need_assistance') %>
+          <%= link_to(t('notices.maintenance.contact_us'), MarketingSite.contact_url) %>
+        </p>
+      </div>
+    </div>
+  </div>
```

Comment:
Would it be possible to reuse the [`_alert.html.erb` partial](https://github.com/18F/identity-idp/blob/master/app/views/shared/_alert.html.erb) here?

```suggestion
  <%= render 'shared/alert', { type: 'warning', class: 'margin-bottom-2' } do %>
    <p>
      <%= t('notices.maintenance.currently_under_maintenance_html',
            finish: l(maintenance_window.finish,
                      format: t('time.formats.event_timestamp_with_zone'))) %>
    </p>
    <p class="margin-bottom-0">
      <%= t('notices.maintenance.need_assistance') %>
      <%= link_to(t('notices.maintenance.contact_us'), MarketingSite.contact_url) %>
    </p>
  <% end %>
```

I guess if we're using `<div>` for `.usa-alert__text`, it won't work to nest the paragraphs. But I think it's something where either we shouldn't be using it that way, or we could update the alert partial to support multiple paragraphs of content like this.

Reply:
Oh duh, I forgot that partial existed 🙈 

Two ideas:
1. I could just add two `<br />`s to be able to reuse the `<p>` (booooo)
2. I could make a local param for `alert_text_tag` that defaults to `<p>` and override it to `div` here?

Reply:
I went for (2) in ce2c96349191898d6e21518fd5845172522877d4, WDYT?

Reply:
Seems reasonable to me. It got me wondering if there's really much effect to using `p` over `div`, as in whether we could just default to `div`. I think there's some value in having the semantics associated with a paragraph, though I'm not sure there's actually any practical difference one way or the other. 

There's also another option where the block usage could assume more advanced HTML, use `div` as default, and assume the developer will pass one (or more) `<p>` tags. The `message` parameter could be reserved for simple strings and use `p` as the default wrapper.

But as I sit here trying to explain it, it sounds like it might be more complicated than it is worthwhile 😄

Reply:
We only use `shared/alert` in 3 places..... let's not make it that complicated yet